apps/api/src/server/migration/migrators/nsip-project-migrator.js

```diff
@@ -28,6 +28,7 @@ export const migrateNsipProjects = async (models) => {
 		await migrateCase(model);
 		await migrateApplicationDetails(model);
 		await migrateRegions(model);
+		await invalidateCaseStatuses(model);
```

Comment:
you could call this function within the `migrateCaseStatus` function, so all the "case status" related DB stuff is within a single fn. But happy either way so approved 👍